app/views/availabilities/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="container-fluid">
+  <h1>Schedule Availability</h1>
+  <%= link_to 'Logout', destroy_user_session_path, method: :delete %>
+  <%= form_tag("/availabilities", method: "post") do %>
```

Comment:
Don't use the URL, use `availabilities_path`. This way if you wish to somehow change the path everything will get updated